avm/res/api-management/service/tests/e2e/defaults/main.test.bicep

```diff
@@ -36,15 +36,13 @@ resource resourceGroup 'Microsoft.Resources/resourceGroups@2021-04-01' = {
 // ============== //
 
 @batchSize(1)
-module testDeployment '../../../main.bicep' = [
-  for iteration in ['init', 'idem']: {
-    scope: resourceGroup
-    name: '${uniqueString(deployment().name, resourceLocation)}-test-${serviceShort}-${iteration}'
-    params: {
-      name: '${namePrefix}${serviceShort}001'
-      location: resourceLocation
-      publisherEmail: 'apimgmt-noreply@mail.windowsazure.com'
-      publisherName: '${namePrefix}-az-amorg-x-001'
-    }
+module testDeployment '../../../main.bicep' = [for iteration in [ 'init', 'idem' ]: {
```

Comment:
Beware, you're not using the latest Bicep version which is why every Bicep file (& JSON file) got re-formatted / updated.

Please upgrade to the latest (check `bicep --version`) and run `Set-AVMModule -Recurse` on the root folder path of this module again :)